Vizzuality/gfw
# Feature/stories layers

![screen shot 2018-11-06 at 19 27 09](https://user-images.githubusercontent.com/20288774/48085237-fea2f980-e1f9-11e8-8238-be23b485b4d8.png)

## Overview

Using the new cluster layer available for leaflet on the layer-manager, this PR adds three new layers to the `explore` section of the map: User stories, Mongabay stories, Earth Journalism Network stories. These layers are accessible through the explore > stories tab.

Features include:
- cluster layers decodes for parsing endpoint data into geoJSON to add to the layer
- simple styles for the layer icons
- updated stories cards for use when an image has a white background
- adding of stories layers to the map from the explore cards config file

## Notes

run `yarn` as layer-manager has been updated

## Testing

- go to /map, open explore tab, select stories tab, add card to map
- click a cluster on the map and check fit bounds works correctly
- click icon for each layer and check popup renders
